core/commands/tour.go

```diff
@@ -64,11 +62,13 @@ func tourRunFunc(req cmds.Request, res cmds.Response) {
 		fmt.Fprintln(&w, err)
 		fmt.Fprintln(&w, "")
 		fprintTourList(&w, tour.TopicID(cfg.Tour.Last))
+		res.SetOutput(bytes.NewReader(w.Bytes()))
 
 		return
 	}
 
 	fprintTourShow(&w, t)
+	res.SetOutput(bytes.NewReader(w.Bytes()))
```

Comment:
Is there a better way to link the ```io.Reader``` and ```io.Writer``` together without having to create a buffer? I tried ```io.Pipe``` but it didn't work.